packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -909,13 +910,17 @@ class _DraggableScrollableSheetScrollPosition extends ScrollPositionWithSingleCo
     }
   }
 
-  bool get _isAtSnapSize {
-    return extent.snapSizes.any((double snapSize) {
+  /// Checks if the sheet's current size is close to a snap size, returning the
+  /// snap size if so; returns null otherwise.
+  double? get _currentSnapSize {
```

Comment:
Nit: I wouldn't make this a getter, to emphasize that there's some calculation that needs to be done.

```dart
double? _getCurrentSnapSize() {
```

Reply:
See: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#getters-feel-faster-than-methods

Reply:
Ah yes you're right, thanks for catching that.